server/db/schema.js

```diff
@@ -72,6 +75,34 @@ const schema = {
         created_at timestamp with time zone DEFAULT now()
       )`,
     },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.POST_SECONDARY_INSTITUTIONS} (
+        id serial primary key,
+        institute_name varchar(200) not null,
+        health_authority varchar(50) not null,
+        available_seats integer not null,
+        postal_code varchar(10) not null
+      )`,
+    },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.COHORTS} (
+        id serial primary key,
+        cohort_name varchar(200) not null,
+        start_date date not null,
+        end_date date not null,
+        cohort_size integer not null,
+        psi_id integer references ${collections.POST_SECONDARY_INSTITUTIONS},
+        participant_id integer not null,
```

Comment:
Is this field needed when `cohort` is linked to `participant` via `cohort_participants`?

Reply:
nah, I don't think it is!